app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -1,6 +1,6 @@
 module Efile
   module Nj
-    class Nj1040 < ::Efile::TaxCalculator
+    class Nj1040Calculator < ::Efile::TaxCalculator
```

Comment:
LOVE this refactor thank you, I've been thinking about meaning to change this for weeks